## This is the texture description for each object
## By default, EHD is used.
## Related pkg: visor
##
## Author: Ferdian Pratama

Header header
int16 size # DEFAULT= 80 (VECTOR SIZE)
int16[] desc # EHD is used